src/LinuxTracing/ThreadStateManager.h

```diff
@@ -86,6 +91,8 @@ class ThreadStateManager {
     // to the runnable state.
     pid_t wakeup_tid;
     pid_t wakeup_pid;
+
+    bool wait_for_callstack;
```

Comment:
I think this should have a comment like the other fields.

Reply:
Done.